docs/modules/ROOT/pages/how-to-guides/Import-code/proc_importing_code.adoc

```diff
@@ -214,7 +214,38 @@ For more information about devfile contents requirements, see the "What is outer
 +
 **Solution:** Select a predefined runtime type from the **Review and configure for deployment** view. Make sure you meet all of the {ProductName} <<repository requirements>> for the runtime you select, or configure your own <<custom build and deployment instructions>>.
 
+=== Using registry.redhat.io base images for builds 
+
+Retrieve the Red Hat Container Registry token and configure your application to use this token. Using this token improves the build process of your application. 
+
+.Procedure
+
+Retrieve the Red Hat Container Registry token by following these steps:
+
+. Go to link:https://access.redhat.com/terms-based-registry/#/[Registry service account]. 
+. Create a new Registry service account by selecting *New Service Account*.  
+. Retrieve the Registry service account token as a “Docker Configuration”:
```

Comment:
lines 227-236: I clicked **New Service Account** and now i'm on the actual **Create a New Registry Service Account** page, and it's not clear to me where a user would do step 3. I see **Name** and **Description** fields ...